modules/shared/src/main/scala/org/tessellation/currency/schema/EstimatedFee.scala

```diff
@@ -1,7 +1,12 @@
 package org.tessellation.currency.schema
 
+import cats.effect.Async
+import cats.implicits.toFunctorOps
```

Comment:
Let's stick to one import style. We do import from cats syntax across the whole codebase, and docs say we're shouldn't mix those two

Reply:
I need to fix my importer, I definitely don't want to use implicit